public/app/assets/javascripts/ReadMore.js

```diff
@@ -0,0 +1,64 @@
+(function (exports) {
+  class ReadMore {
```

Comment:
Should be renamed `ReadMoreNotes` given the readmore parallel of more facets.

Reply:
This is still not renamed, I think it is a good idea to actually rename it...

Reply:
renamed throughout the files